src/main.cpp

```diff
@@ -517,8 +520,10 @@ void dump_document(PDFDoc *doc, const Options &options) {
 }
 
 BaseStream *open_file(const std::string filename) {
-  GooString goo_filename(filename.c_str());
-  auto file = GooFile::open(&goo_filename);
+  // Original
+  // GooString goo_filename(filename.c_str());
+  // auto file = GooFile::open(&goo_filename);
```

Comment:
This should be removed, but I'll do that in another PR as I've tested with the code as it is in this current PR, and the SHA256 of the binary will change.